### PR TITLE
Signup Page complete.

### LINKED ISSUE #72: Sign up
component styling

### LINKED ISSUE #73: Sign Up
functionalty
redux for user

### PR DESCRIPTION
**Pull request by khaldoon9kh**
Sign up Page is completed with redux.

Page:
image

Alert:
image
image
image

### CODE REVIEW

File: src/components/NavBar/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -60,13 +60,20 @@ exports[`navbar renders correctly 1`] = `
     >
       homePage.navbar.signIn
     </button>
-    <button
-      className="shadow-none mr-1 btn btn-outline-info"
-      disabled={false}
-      type="button"
+    <a
+      aria-current={null}
+      className="nav-link"
```
Comment (fadime-ozdemir) [inline src/components/NavBar/__snapshots__/index.test.jsx.snap:65]: is this a bootstrap class?

File: src/components/NavBar/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -60,13 +60,20 @@ exports[`navbar renders correctly 1`] = `
...
+      href="/signup"
+      onClick={[Function]}
```
Comment (fadime-ozdemir) [inline src/components/NavBar/__snapshots__/index.test.jsx.snap:67]: why do you call onClick like this [Function]?

File: src/components/SignUp/index.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  --title-color: #002736;
+  --subTitle-color: #000000;
+  --background-color: #fc9e01;
+  --text-color: #eff4f6;
+  --filter-shadow-color: #b3b3b3;
+  --link-color: #0641af;
+}
+
+.signupContainer {
+  display: flex;
```
Comment (fadime-ozdemir) [inline src/components/SignUp/index.css]: we can use the bootstrap class instead of display flex

File: src/components/SignUp/index.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.signupHeader {
+  color: #6c567b;
```
Comment (fadime-ozdemir) [inline src/components/SignUp/index.css]: you can add it to the color variables, I think you forgot

File: src/components/SignUp/index.jsx [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+      </Row>
+      <Form onSubmit={handleSignup} className="form">
+        <Form.Row>
+          <Form.Group as={Col} controlId="formGridFirstName" className="mt-3">
+            <Form.Control
+              required
+              type="name"
+              placeholder={t("signup.firstName")}
+              value={signupInfo.firstName}
+              onChange={handleChange("firstName")}
```
Comment (fadime-ozdemir) [inline src/components/SignUp/index.jsx]: Nice, so we don't need to pass the event directly, do we?

File: src/firebaseConfig.js [hunk 6]
```diff
@@ -12,5 +12,6 @@ const firebaseConfig = {
 // Initialize Firebase
 firebase.initializeApp(firebaseConfig);
 
-export default firebase.firestore();
-export const Auth = firebase.auth;
+export const fireStore = firebase.firestore();
```
Comment (fadime-ozdemir) [inline src/firebaseConfig.js]: maybe you need import one of these constants as default. The tests aren't passing

**Review by fadime-ozdemir — CHANGES_REQUESTED** [review]

File: src/components/NavBar/__snapshots__/index.test.jsx.snap (hunk 2 above)
Comment (khaldoon9kh) [reply src/components/NavBar/__snapshots__/index.test.jsx.snap:67]: I wrapped the button with NavLink to take us to the SignUp page. I did not want to change it to not mess with the NavBar styling.

Comment (fadime-ozdemir) [reply src/components/NavBar/__snapshots__/index.test.jsx.snap:67]: I notice now that I was reviewing the index.test.jsx.snap. Don't take in consideration what I wrote

File: src/components/NavBar/__snapshots__/index.test.jsx.snap (hunk 1 above)
Comment (fadime-ozdemir) [reply src/components/NavBar/__snapshots__/index.test.jsx.snap:65]: I notice now that I was reviewing the index.test.jsx.snap. Don't take in consideration what I wrote

File: src/components/SignUp/index.css (hunk 3 above)
Comment (khaldoon9kh) [reply src/components/SignUp/index.css]: edited.

File: src/components/SignUp/index.css (hunk 4 above)
Comment (khaldoon9kh) [reply src/components/SignUp/index.css]: revised

File: src/components/SignUp/index.jsx (hunk 5 above)
Comment (khaldoon9kh) [reply src/components/SignUp/index.jsx]: yes we don't.

File: src/firebaseConfig.js (hunk 6 above)
Comment (khaldoon9kh) [reply src/firebaseConfig.js]: revised

**Comment by khaldoon9kh** [comment]
PTAL.

File: src/locales/en.json [hunk 7]
```diff
@@ -93,6 +93,19 @@
       }
     }
   },
+  "signup": {
+    "firstName": "First Name",
+    "lastName": "Last Name",
+    "email": "email",
```
Comment (halitbatur) [inline src/locales/en.json]: capitalization

**Review by halitbatur — COMMENTED** [review]
really familiar code ;) fix this one and its merged

**Review by halitbatur — APPROVED** [review]

File: src/locales/en.json (hunk 7 above)
Comment (khaldoon9kh) [reply src/locales/en.json]: The capitalization is for the object value only... I did not understand what is the error?

Comment (khaldoon9kh) [reply src/locales/en.json]: Revised

**Comment by khaldoon9kh** [comment]
PTAL

**Review by senzeynep — APPROVED** [review]
LGTM

**Review by fadime-ozdemir — APPROVED** [review]
LGTM

**Pull request merged**